### PR TITLE
Enable Cloudwatch insights for performance tests

### PR DESCRIPTION
**Pull request by jaydeokar**
**What type of PR is this?**
automation

**Which issue does this PR fix**:
N/A

**What does this PR do / Why do we need it**:
This collects pod memory, CPU and other metrics through cloudwatchagent and pushes it to cloudwatch insights. It will collect metrics every 30s which gives us granular information about resource usage during performance tests

**If an issue # is not available please add repro steps and logs from IPAMD/CNI showing the issue**:
N/A

**Testing done on this change**:
Yes

**Automation added to e2e**:
N/A

**Will this PR introduce any new dependencies?**:
N/A

**Will this break upgrades or downgrades. Has updating a running cluster been tested?**:
N/A

**Does this change require updates to the CNI daemonset config files to work?**:
N/A

**Does this PR introduce any user-facing change?**:
N/A

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jdn5126 — COMMENTED** [review]
Change LGTM. Do we need the metrics-server if we have CloudWatch logs?

**Comment by jaydeokar** [comment]
> Change LGTM. Do we need the metrics-server if we have CloudWatch logs?

No we don't. I kept it still as we don't want to remove the existing metrics collection flow (s3 bucket). We can clean that up once we have the right dashboards and metrics in place with CW

File: scripts/test/config/perf-cluster.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@
 ---
 apiVersion: eksctl.io/v1alpha5
 kind: ClusterConfig
+iam:
```
Comment (orsenthil) [inline scripts/test/config/perf-cluster.yml:5]: Is this more for security / improvement or is it required?

Comment (jaydeokar) [reply scripts/test/config/perf-cluster.yml:5]: I wanted to use IRSA for creating the SA/permissions for cloudwatch agent. Unless we enable this, IRSA doesn't work

**Review by jdn5126 — APPROVED** [review]

**Pull request merged**